jetson/percep/main.cpp

```diff
@@ -130,6 +130,14 @@ int main() {
         DoubleTrack(&detector.getAlvarParams()->maxErroneousBitsInBorderRate).setup("maxErroneousBitsInBorderRate", "ARUCO Debug", 1);
         DoubleTrack(&detector.getAlvarParams()->minOtsuStdDev).setup("minOtsuStdDev", "ARUCO Debug", 20);
         DoubleTrack(&detector.getAlvarParams()->errorCorrectionRate).setup("errorCorrectionRate", "ARUCO Debug", 1);
+
+        std::string const gstLaunch = "appsrc ! video/x-raw, format=BGR !"
+                                    "queue ! videoconvert ! x264enc tune=zerolatency bitrate=1000000 speed-preset=superfast !"
+                                    "rtph264pay ! udpsink host=127.0.0.1 port=5000";
```

Comment:
can you change it to port 5002 or something? in auton 5000 and 5001 are being used by the regular usb cameras